### DISCUSSION TITLE
New: How is this digital content created?  "With an AI generator"

### DISCUSSION THREAD

**Discussion by slider1234** [etsy/open-api#1269] [Q&A]
This shows as an option for listings when you manually look at your product, but there is no documentation to be found to use it in the API 

**How is this digital content created?**
1. It’s designed and created entirely by me. 
2. With an AI generator

This was just intriduced as a new option and being told it has to be selected, but no documentation for it anywhere i can find.
Any help would be much appreciated.

**Comment by Unicorn2458** [comment]
Did you ever get an answer to this? I have the same issue

**Comment by EfeErdem7** [comment]
I have the same issue. I can't find anything about this in the docs either.
